airflow/api_fastapi/core_api/routes/public/connections.py

```diff
@@ -180,3 +186,44 @@ async def patch_connection(
     for key, val in data.items():
         setattr(connection, key, val)
     return ConnectionResponse.model_validate(connection, from_attributes=True)
+
+
+@connections_router.post(
+    "/test",
+    responses=create_openapi_http_exception_doc(
+        [
+            status.HTTP_401_UNAUTHORIZED,
+            status.HTTP_403_FORBIDDEN,
+        ]
+    ),
+)
+async def test_connection(
```

Comment:
New convention, function must not be declared async.

More details here: 
https://github.com/apache/airflow/pull/43797

Can you remove it then we can merge ?

Reply:
I’ll definitely make those changes. I came across this PR this morning and realized I missed it yesterday while pushing the documentation updates. Thanks for pointing it out!